tox.ini

```diff
@@ -25,6 +27,8 @@ deps=
     django111: django>=1.11,<2.0
     django20: django>=2.0,<2.1
     django21: django>=2.1,<2.2
+    django22: django>=2.2,<3.0
+    django30: django>=3.0,<3.1
```

Comment:
Since Django 3.1 has been released in August, should it be included?

Reply:
I completely forgot about this PR (I have opened it in March). Will add 3.1 in a sec :)

Reply:
Will this be merged now you're reviewing it @aleyan ?

Reply:
Thank you @D3X . @simkimsia Would love to merge this, but unfortunately I don't have access, thats up to @jmcarp .